asdf_astropy/resources/manifests/astropy-1.0.0.yaml

```diff
@@ -28,6 +28,16 @@ tags:
     be accessible in its proper form in the ASDF file.
 
     Only image and binary table extensions are supported.
+- tag_uri: tag:astropy.org:astropy/fits/fitswcs-1.0.0
```

Comment:
Would you add this to a new manifest `astropy-1.1.0.yaml` and register it as a new extension?

Reply:
Sure

Reply:
`astropy-1.1.0.yaml` manifest already exists. Do I need to add it to this manifest?

Reply:
Yeah a new manifest is needed (I forgot this already has an 1.1.0).

Things are a bit more complicated as 1.2.0 exists but is only ASDF standard 1.6.0 compliant.

Would you add a new one (1.3.0) based off of the 1.1.0 one (which uses the current ASDF standard 1.5.0). I'll sort out the 1.6.0 issues after this PR is in.